src/App.js

```diff
@@ -0,0 +1,17 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Weather from './component/weather';
+import CityDetail from './component/cityDetail';
+
+function App() {
+  return (
+    <div className="App flex flex-col">
+      <Routes>
+        <Route index element={<Weather />} />
+        <Route path="/city/:cityName" element={<CityDetail />} />
+      </Routes>
+    </div>
```

Comment:
- [ ] Kindly make sure to follow the design's guidelines when it comes to fonts and font styles, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.) requirement.

Reply:
> - [ ] Based on the previous reviewer feedback, Please ensure that you adhere to the design's specifications regarding fonts and font styles, as outlined in [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.) requirement. This enhances the overall user experience and makes your application or website appear more polished and professional.😊
> 
> Provided Design:
> 
> ![cot](https://github.com/sheezu900/React-Capstone-Project/assets/40009354/72ae04b6-438b-4f1e-9ee2-c3356694df26)
> 
> Your Design:
> 
> ![cot1](https://github.com/sheezu900/React-Capstone-Project/assets/40009354/3901ad21-4019-4508-a31d-7f0457f5fd00)
>

Reply:
- [Optional] Dear @sheezu900, I believe your design could still use some adjustments to be aligned with the original design better. Please consider following the original design closely. This includes color, layout composition, font type, and more. Remember as a frontend engineer we are expected to deliver the identical design.